### PR TITLE
Add e2e tests for GCP CLI tool

### PR DESCRIPTION
**Pull request by giovannt0**
As part of addressing #164, starting with GCP. The tests mostly follow what we already have in place for e2e tests, the only difference being that operations are delegated to the CLI tool in place of the `forensics` module.

### CODE REVIEW

**Comment by Fryyyyy** [comment]
Why are these called *bash ?

**Comment by giovannt0** [comment]
@Fryyyyy because I'm dumb and forgot to change the files name (I was initially gonna do some bash stuff hence the name, but changed my mind). Fixed!

File: tests/providers/gcp/e2e_cli.py [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+    logger.info('Deleting analysis instance: {0:s}.'.format(
+        analysis_vm.name))
+    gce_instance_client = gcp_client.GceApi().instances()
+    request = gce_instance_client.delete(
+        project=project.project_id, zone=zone, instance=analysis_vm.name)
+    response = request.execute()
+    try:
+      gcp_client.BlockOperation(response)
+    except HttpError:
+      # BlockOperation triggers a while(True) loop that checks on the
```
Comment (Fryyyyy) [inline tests/providers/gcp/e2e_cli.py:237]: Should we have a bug/issue open that tracks this and changes it to a tighter While loop (on a condition that changes when the operation has finished?)

**Review by Fryyyyy — APPROVED** [review]

File: tests/providers/gcp/e2e_cli.py (hunk 1 above)
Comment (giovannt0) [reply tests/providers/gcp/e2e_cli.py:237]: Agreed, I'll open one, thanks!

**Pull request merged**